4-star-nosed_mole/api_definitions/rest/ucs.yaml

```diff
@@ -24,10 +24,14 @@ paths:
                 $ref: "#/components/schemas/FileMetadata"
         "403":
           description: |
-            The user is not registered as a Data Submitter for the
-            corresponding file.
+            Exceptions by ID:
+              - userNotRegisteredFileAccess: The user is not registered
+                as a Data Submitter for the corresponding file.
         "404":
-          description: The file with the given ID has not (yet) been registered for upload.
+          description: |
+            Exceptions by ID:
+             - fileNotRegisteredAccess: The file with the given ID has
```

Comment:
```suggestion
             - fileNotRegisteredYet: The file with the given ID has
```

Reply:
Not sure about this one. There is no way to have knowledge if the registration is coming/ongoing at that point in the code, right? Then it might be better to just have that as `fileNoteRegisterd`

Reply:
Yes that makes sense